packages/metals-vscode/src/testExplorer/addTestSuites.ts

```diff
@@ -72,16 +81,37 @@ export function addTestSuite(
  */
 function getOrCreateBuildTargetItem(
   testController: vscode.TestController,
-  targetName: TargetName,
-  targetUri: TargetUri
+  workspaceFolderItem: vscode.TestItem,
+  targetName: TargetName
 ): vscode.TestItem {
-  const buildTarget = testController.items.get(targetName);
+  const buildTarget = workspaceFolderItem.children.get(targetName);
   if (buildTarget) {
     return buildTarget;
   }
 
   const createdNode = testController.createTestItem(targetName, targetName);
```

Comment:
Should prefix target's test item id with workspace? Or it isn't necessary because id has to be unique only in a given subtree and if we're child of workspace then we're already guaranteed to be unique?

Reply:
```
@param id Identifier for the TestItem. The test item's ID must be in the {@link TestItemCollection} it's added to
where
TestItemCollection is a collection of test items, found in TestItem.children and TestController.items.
```
So it only has to be unique among its siblings and we always access it by first accessing the parent (workspace folder).